requirements/edx/base.in

```diff
@@ -78,7 +78,9 @@ edx-celeryutils
 edx-completion
 edx-django-release-util             # Release utils for the edx release pipeline
 edx-django-sites-extensions
-edx-django-utils
+# edx-django-utils 3.8.0 renames metric -> attribute and deprecates the old
+# methods; edx-platform calls the new names
+edx-django-utils>=3.8.0
```

Comment:
Nit: I prefer something shorter like https://github.com/edx/edx-drf-extensions/blob/master/requirements/base.in#L7, but either works.

Reply:
That seems nicely terse. May still keep it on a separate line since we conventionally use the same-line comment to describe the purpose/use of the dep.

Reply:
👍